src/transactions/feed/JumpstartFeedItem.tsx

```diff
@@ -0,0 +1,127 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { hideHomeBalancesSelector } from 'src/app/selectors'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { useSelector } from 'src/redux/hooks'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import { useTokenInfo } from 'src/tokens/hooks'
+import TransactionFeedItemImage from 'src/transactions/feed/TransactionFeedItemImage'
+import { TokenTransactionTypeV2, TokenTransfer } from 'src/transactions/types'
+
+interface Props {
+  transfer: TokenTransfer
+}
+
+function JumpstartFeedItem({ transfer }: Props) {
+  const { t } = useTranslation()
+  const { amount, type } = transfer
+
+  const openTransferDetails = () => {
+    navigate(Screens.TransactionDetailsScreen, { transaction: transfer })
```

Comment:
since we already know that this transaction is a jumpstart transaction here, can we add this information as a nav parameter? then the TransactionDetailsScreen would already receive this information as a boolean, and wouldn't need to call `isJumpstartTransaction` at all. (and then `isJumpstartTransaction` would only need to be called once in the TransactionFeed, and perhaps can live as an unexported function in that file.